classes/services/PKPContextService.inc.php

```diff
@@ -341,6 +341,33 @@ public function validate($action, $props, $allowedLocales, $primaryLocale) {
 			}
 		});
 
+		// Transforming copySubmissionAckAddress from CSV to array
+		$validator->after(function ($validator) use ($props) {
+			if (!isset($props['copySubmissionAckAddress'])) {
+				return;
+			}
+
+			$emails = explode(',', $props['copySubmissionAckAddress']);
+
+			if ($emails === []) {
+				return;
+			}
+
+			foreach ($emails as $currentEmail) {
+				$value = trim($currentEmail);
+
+				$emailValidator = \ValidatorFactory::make(
+					['value' => $value],
+					['value' => ['email_or_localhost']]
+				);
+
+				if ($emailValidator->fails()) {
+					$validator->errors()->add('copySubmissionAckAddress', __('manager.setup.notifications.copySubmissionAckAddress.invalid'));
+					continue;
```

Comment:
Let's use `break` instead of `continue`. If more than one email address is invalid, the error message would appear more than once on the same field.